executor/modes/local/progress_reporter.py

```
@@ -0,0 +1,220 @@
+                {
+                    "task_id": self.task_id,
+                    "subtask_id": self.subtask_id,
+                    "task_title": self.task_title,
+                    "subtask_title": self.subtask_title,
+                    "status": status,
+                    "result": result,
+                    "message": message,
+                },
+            )
+            logger.info(f"Result reported: task_id={self.task_id}, status={status}")
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)